.github/workflows/change_task_count.yaml

```diff
@@ -0,0 +1,35 @@
+name: Change Task Count
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: What environment to change the task count for
+        options:
+          - dev
+          - staging
```

Comment:
use a choice rather than having us enter the string. it should create a drop down on the github action ui.